domains/creed.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "M-creed",
+        "email": "marwankalosha005@gmail.com"
+    },
+    "record": {
+        "CNAME": "m-creed.github.io"
```

Comment:
cant access website (404 page)

Reply:
his website is accessible at https://m-creed.github.io/Portfolio-Template/